public/pages/EditFeatures/containers/SampleAnomalies.tsx

```diff
@@ -235,8 +235,9 @@ export function SampleAnomalies(props: SampleAnomaliesProps) {
                 onZoomRangeChange={handleZoomChange}
                 isLoading={isLoading}
                 dateRange={dateRange}
-                detectorId={props.detector.id}
-                detectorName={props.detector.name}
+                // detectorId={props.detector.id}
+                // detectorName={props.detector.name}
+                detector={props.detector}
```

Comment:
minor: can we remove these commented out lines altogether?

Reply:
good catch. will remove them.